docs/dev_guide/diagnostic_ref/dpct1073.rst

```diff
@@ -30,36 +30,49 @@ For example, this original CUDA\* code:
 .. code-block:: cpp
    :linenos:
   
+   void foo(){
      CUDA_ARRAY_DESCRIPTOR *desc_ptr;
      CUDA_ARRAY_DESCRIPTOR desc;
      desc_ptr = &desc;
-  
-     Cuarray arr;
-     cuArrayCreate(&arr, desc_ptr)
+   
+     CUarray arr;
+     cuArrayCreate(&arr, desc_ptr);
+   }
 
 results in the following migrated SYCL\* code:
 
 .. code-block:: cpp
    :linenos:
-  
+
+   void foo(){

```

Comment:
Please update the sample code with init part for desc.

Reply:
done